### DISCUSSION TITLE
Your QR for best camera settings

### DISCUSSION THREAD

**Discussion by edaz3** [gopro/labs#1042] [Show and tell]
Hi everyone! Can you show your best QR for gopro? I have gopro 12 black but I don't know the best settings for video. I'm sure many people have QR's for video at night, afternoon or for timelapse.

**Comment by dnewman-gpsw** [comment]
This is no best for those conditions, as it would all depend on what you are shooting, and how you are using the video. So you need to be more specific, and likely craft you own setup.
